examples/wifi/expanded/requests_wifi_file_upload.py

```diff
@@ -12,10 +12,10 @@
 ssl_context = adafruit_connection_manager.get_radio_ssl_context(wifi.radio)
 requests = adafruit_requests.Session(pool, ssl_context)
 
-with open("raspi_snip.png", "rb") as file_handle:
+with open("requests_wifi_file_upload_image.png", "rb") as file_handle:
```

Comment:
file added had this name

Reply:
yep, changed that at the last moment before I committed it, looks like I forgot to update the string in the file. Thank you.